code/modules/projectiles/projectile_parent.dm

```diff
@@ -766,6 +766,7 @@ datum/projectile/tele_bolt
 
 datum/projectile/rad_bolt
 	impact_range = 0
+	ie_type = null // Probably not the best way to make sure these don't generate an impact effect but oh well.
```

Comment:
put this on the proper projectile datum definition - all this stuff here should be cleaned up eventually